app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,37 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+  def create
+    if user_signed_in? && current_user.id == params[:user_id] && request.post?
+      Rent.create(user_id: params[:user_id], book_id: params[:book_id],
+                  from: params[:from], to: params[:to], returned_at: params[:returned_at])
+    else
+      render json: {}, status: 401
+    end
+  end
+
+  def index
+    if user_signed_in? && current_user.id == params[:user_id] && request.get?
```

Comment:
If the request isn't a get it won't get here because there will be no route pointing at rents#index. No need to check for that.
Also, the same condition is repeated in 3 different places, delegate that to a method (that method will disappear when you use Pundit though). Duplicate code is evil